client/me/purchases/add-new-payment-method/index.jsx

```diff
@@ -56,6 +58,10 @@ function AddNewPaymentMethod() {
 
 	const title = `${ titles.activeUpgrades } › ${ addPaymentMethodTitle }`;
 
+	if ( isWcMobileApp() ) {
+		reduxDispatch( hideMasterbar() );
```

Comment:
I just need to figure out if we need to explicitly call `showMasterbar` at some point, to make the `masterbar` visible again on other pages. Not sure yet if it's needed, though.

Reply:
@hichamboushaba on second thought, should we be hiding the `masterbar` on **_all_** pages, as was done in https://github.com/Automattic/wp-calypso/pull/54189, or is the goal to only hide it on `/me/purchases/add-credit-card` and nowhere else?

Reply:
@guarani our goal was to hide everywhere in case we need to show the web UI elsewhere, though the immediate need is only for `/me/purchases/add-credit-card`.

Reply:
Sounds good! I think it simplifies the implementation so I made the change to hide the `masterbar` on all pages when viewed from the WC apps.

Reply:
Making this as resolved.